source/Hazard.cpp

```diff
@@ -34,7 +34,7 @@ void Hazard::Load(const DataNode &node)
 	{
 		const string &key = child.Token(0);
 		if(key == "weapon")
-			LoadWeapon(child);
+			Weapon::Load(child);
```

Comment:
Should we also change Hazard to no longer inherit from weapon (in a separate PR)? I can imagine that some of the arguments for not letting outfits inherit from weapon could also be relevant for hazards.

Reply:
perhaps